packages/modules/viewer-react/package.json

```diff
@@ -88,6 +88,7 @@
     "@itwin/core-geometry": "^3.2.0",
     "@itwin/core-react": "^3.2.0",
     "@itwin/imodels-access-frontend": "^1.0.1",
+    "@itwin/imodels-client-management": "^1.0.1",
```

Comment:
sigh, good catch. technically new peers are breaking, but no one should be directly using this pkg, (since its supposed to be consumed directly in desktop & web viewer), so i'm more ok adding this as a peer, but lets make sure this is a minor bump?

Reply:
This probably could be considered a non-breaking change since _@itwin/imodels-client-management_ was always required just not specified.

> lets make sure this is a minor bump?

Not completely sure what you mean by this. There is no other breaking changes to packages in this PR, so I guess this could be even a patch bump.